docker/maven_build/pom/api/pom.xml

```diff
@@ -1,10 +1,8 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>gov.nasa.pds</groupId>
 	<artifactId>registry-api-service</artifactId>
-	<version>0.1.0</version>
+	<version>0.4.0-SNAPSHOT</version>
```

Comment:
@tdddblog for these POM files that are out there, are these just included in this repo as a placeholder and then the new version is  pulled from the branch on the repo?

Reply:
For now I just manually copied poms from API, Registry Manager, Harvest, and Commons projects. Also had to add few "extra" poms to pull surefire junit dependencies (Harvest and API use different versions). For some reason main poms don't pull junit dependencies.